src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -435,6 +436,7 @@ def transform_request(self,
         :param pagination: Pagination to be used for the API
         :param post_filter: Flag to indicate whether to do a post_filter
         call instead of the regular query.
+        :param include_file_urls: Show file url field in request
```

Comment:
In English, *URL* is an acronym and needs to be all-caps. For variables `url` is fine.